test/metadata.js

```diff
@@ -31,9 +31,7 @@ describe('Metadata source plugin', () => {
 
   it('initializes a timer with the set interval', (done) => {
     this.m.on('update', () => {
-      const status = this.m.status();
-
-      status.interval.should.have.keys(['_called', '_idleNext', '_idlePrev', '_idleStart', '_idleTimeout',
+      this.m._timer.should.have.keys(['_called', '_idleNext', '_idlePrev', '_idleStart', '_idleTimeout',
```

Comment:
No, `this.m` is the metadata plugin instance created on line 16. Because I changed the output of `Metadata#status()` to expose the interval rather than the timer itself, we have to dig into "private" properties and test whether the `_timer` object has keys consistent with a timer.